### PR TITLE
Similarity list bugfix

### PR DESCRIPTION
**Pull request by efelmandar**
## Status
- [ ] In Progress
- [ ] Ready
- [x] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-7614)

## Description
This fixes a bug with handling lists of strings

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import difflib
 
 
-def stringSimilarity(firstString: str, second_string: str, similarityThreshold: float):
+def stringSimilarity(firstString: str, secondString: str, similarityThreshold: float):
```
Comment (eyalpalo) [inline Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity.py:6]: Can you please change all vars from camel case to using underscore as
firstString -> first_string, etc.

File: Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity.py [hunk 2]
```diff
@@ -25,26 +25,31 @@ def stringSimilarity(firstString: str, second_string: str, similarityThreshold:
...
-    second_string = demisto.getArg('string_B')
-    commandResults = stringSimilarity(first_string, second_string, similarity_threshold)
+    similarityThreshold = demisto.getArg('similarity_threshold')
+    firstString = demisto.getArg('string_A')
+    secondString = demisto.getArg('string_B')
+
+    try:
+        results = stringSimilarity(firstString, secondString, similarityThreshold)
+
+        commandResults = CommandResults("StringSimilarity", ["StringA", "StringB"], results)
```
Comment (eyalpalo) [inline Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity.py:48]: We can move the CommandResults to be in the stringSimilarity itself.

File: Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity_test.py [hunk 3]
```diff
@@ -21,12 +21,14 @@
...
-    try:
-        result = stringSimilarity(first_string, second_string, similarity_threshold)
-        assert result.outputs == expected_result
-    except ValueError as e:
-        assert "No similarity score calculated" in str(e)
+def test_string_similarity(first_string, second_string, similarity_threshold, expected_result, mocker, capfd):
+    with capfd.disabled():
+        try:
+            result = stringSimilarity(first_string, second_string, similarity_threshold)
+            print(result)
```
Comment (eyalpalo) [inline Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity_test.py:28]: Please remove print statement.

**Review by eyalpalo — CHANGES_REQUESTED** [review]
Nice!

**Review by eyalpalo — APPROVED** [review]

**Comment by DeanArbel** [comment]
Validate fails due to `BC103` - the change is inteneded.

**Pull request merged**